Makefile

```diff
@@ -0,0 +1,110 @@
+help: ##@other Show this help
+	@perl -e '$(HELP_FUN)' $(MAKEFILE_LIST)
+
+# This is a code for automatic help generator.
+# It supports ANSI colors and categories.
+# To add new item into help output, simply add comments
+# starting with '##'. To add category, use @category.
+GREEN  := $(shell tput -Txterm setaf 2)
+WHITE  := $(shell tput -Txterm setaf 7)
+YELLOW := $(shell tput -Txterm setaf 3)
+RESET  := $(shell tput -Txterm sgr0)
+HELP_FUN = \
+		   %help; \
+		   while(<>) { push @{$$help{$$2 // 'options'}}, [$$1, $$3] if /^([a-zA-Z\-]+)\s*:.*\#\#(?:@([a-zA-Z\-]+))?\s(.*)$$/ }; \
+		   print "Usage: make [target]\n\n"; \
+		   for (sort keys %help) { \
+			   print "${WHITE}$$_:${RESET}\n"; \
+			   for (@{$$help{$$_}}) { \
+				   $$sep = " " x (32 - length $$_->[0]); \
+				   print "  ${YELLOW}$$_->[0]${RESET}$$sep${GREEN}$$_->[1]${RESET}\n"; \
+			   }; \
+			   print "\n"; \
+		   }
+
+# Main targets
+
+prepare: ##@prepare Install dependencies and prepare workspace
+	lein deps
+	npm install
+	./re-natal deps
+	./re-natal use-figwheel
+	lein re-frisk use-re-natal 
+	./re-natal enable-source-maps
+
+prepare-ios: prepare ##@prepare Install iOS specific dependencies
+	cd ios && pod install && cd ..
+
+#----------------
+# Release builds
+#----------------
+release: release-android release-ios ##@build build release for Android and iOS
+
+release-android: prod-build ##@build build release for Android 
+	react-native run-android --variant=release
+
+release-ios: prod-build ##@build build release for iOS release
+	@echo "Build in XCode, see https://wiki.status.im/TBD for instructions"
+
+#----------------
+# Dev builds
+#----------------
+dev-android-real: ##@dev build for Android real device
+	./re-natal use-android-device real
+	./re-natal use-figwheel
+
+dev-android-avd: ##@dev build for Android AVD simulator
+	./re-natal use-android-device avd
+	./re-natal use-figwheel
+
+dev-android-genymotion: ##@dev build for Android Genymotion simulator
+	./re-natal use-android-device genymotion
+	./re-natal use-figwheel
+
+dev-ios-real: prod-build ##@dev build for iOS real device
+	./re-natal use-ios-device real
+	./re-natal use-figwheel
+
+dev-ios-simulator: prod-build ##@dev build for iOS simulator
+	./re-natal use-ios-device simulator
+	./re-natal use-figwheel
+
+prod-build:
+	lein prod-build
+
+#--------------
+# REPL
+# -------------
+
+repl: ##@repl Start REPL for iOS and Android
+	BUILD_IDS="ios,android" lein repl
+
+repl-ios: ##@repl Start REPL for iOS
+	BUILD_IDS="ios" lein repl
+
+repl-android: ##@repl Start REPL for Android
+	BUILD_IDS="android" lein repl
+
+#--------------
+# Run
+# -------------
+run-android: ##@run Run Android build
+	react-native run-android
+
+run-ios: ##@run Run iOS build
+	react-native run-ios
+
+#--------------
+# Other
+#--------------
+react-native: ##@other Start react native packager
```

Comment:
This will need a `.PHONY: react-native` directive because `react-native` is an actual file, otherwise running `make react-native` results in:
```
make: 'react-native' is up to date.
```

I don't have perms to make this change.

Reply:
Would also be nice to have a `test` target which includes `lein test-cljs && lein prod-build` and whatever canonical tests can be run on the system (likely what devs and CI/CT runs)

Reply:
@charlesg3 oh, nice catch, thanks. There was no directory `react-native` at the beginning, but now I have it as well. Done.

As for the test — I believe it's better to do in a separate PR. Actually, I encourage Clojure Team to add/update/rename Makefile targets as they wish, because they have more knowledge on how to work with this codebase. I basically copy-pasted commands from wiki.